docker/pg_tests/scripts/configs/config_test_config.json

```diff
@@ -0,0 +1,13 @@
+{"PGSSLMODE": "allow",
```

Comment:
If I'm not mistaken. You place "{ }" in config files and also wrap these configs with "{ }".

Reply:
If you don't wrap these files. It would be better to have similar files without brackets and wrap all of them in tests.